src/gsy_e/gsy_e_core/live_events.py

```diff
@@ -43,8 +46,31 @@ def __repr__(self):
                f"params({self.area_representation}))>"
 
 
+class CreateAreaEventCoefficient(CreateAreaEvent):
+    """Event that creates a new area on the area representation tree."""
```

Comment:
NIT:
"Event that creates a new area on the area representation tree for SCM simulations"